boards/arm/nucleo_h7a3zi_q/nucleo_h7a3zi_q_defconfig

```diff
@@ -25,3 +25,6 @@ CONFIG_GPIO=y
 
 # Enable clocks
 CONFIG_CLOCK_CONTROL=y
+
+# Enable SMPS
+CONFIG_POWER_SUPPLY_SMPS=y
```

Comment:
This breaks bisectability in commit: 005b0858b95be96aab6383e2df32b5704658400a
```
$ git checkout 005b0858b95be96aab6383e2df32b5704658400a
$ cmake -GNinja -DBOARD=nucleo_h7a3zi_q -Bbuild samples/hello_world
...
$ ninja -Cbuild
...
[77/149] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32h7/soc_m7.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32h7/soc_m7.c.obj 
ccache /opt/gcc-arm-none-eabi-9-2020-q2-update/bin/arm-none-eabi-gcc -DBUILD_VERSION=v2.7.99-1511-g005b0858b95b -DCORE_CM7 -DHSE_VALUE=8000000 -DKERNEL -DSTM32H7A3xx -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I../kernel/include -I../arch/arm/include -I../include -Izephyr/include/generated -I../soc/arm/st_stm32/stm32h7 -I../drivers -I../soc/arm/st_stm32/common -I/projects/github/ncs/modules/hal/cmsis/CMSIS/Core/Include -I/projects/github/ncs/modules/hal/stm32/stm32cube/stm32h7xx/soc -I/projects/github/ncs/modules/hal/stm32/stm32cube/stm32h7xx/drivers/include -I/projects/github/ncs/modules/hal/stm32/stm32cube/stm32h7xx/drivers/include/Legacy -I/projects/github/ncs/modules/hal/stm32/stm32cube/common_ll/include -isystem ../lib/libc/minimal/include -isystem /opt/gcc-arm-none-eabi-9-2020-q2-update/bin/../lib/gcc/arm-none-eabi/9.3.1/include -isystem /opt/gcc-arm-none-eabi-9-2020-q2-update/bin/../lib/gcc/arm-none-eabi/9.3.1/include-fixed -Os -imacros /projects/github/ncs/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m7 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /projects/github/ncs/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-address-of-packed-member -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/projects/github/ncs/zephyr/samples/hello_world=CMAKE_SOURCE_DIR -fmacro-prefix-map=/projects/github/ncs/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/projects/github/ncs=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32h7/soc_m7.c.obj -MF zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32h7/soc_m7.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32h7/soc_m7.c.obj -c /projects/github/ncs/zephyr/soc/arm/st_stm32/stm32h7/soc_m7.c
/projects/github/ncs/zephyr/soc/arm/st_stm32/stm32h7/soc_m7.c: In function 'stm32h7_init':
/projects/github/ncs/zephyr/soc/arm/st_stm32/stm32h7/soc_m7.c:86:2: error: #error Unsupported configuration: Selected SoC do not support SMPS
   86 | #error Unsupported configuration: Selected SoC do not support SMPS
      |  ^~~~~
[94/149] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/cortex_m_systick.c.obj
ninja: build stopped: subcommand failed.
```

Please introduce `CONFIG_POWER_SUPPLY_SMPS=y` together with the SoC supporting this, or in a following commit.

For example by `rebase -i` and re-order the commits.

Reply:
Ok re-ordered.